pip/src/interpreter.rs

```diff
@@ -494,6 +576,83 @@ impl Interpreter {
     }
 }
 
+/// Finds any Q# type recursively that does not support interop with Python, meaning our code cannot convert it back and forth
+/// across the interop boundary.
+fn first_unsupport_interop_ty(ty: &Ty) -> Option<&Ty> {
```

Comment:
is this a typo? `first_unsupport_interop_ty`

Reply:
Whoops, yes, will fix.